### PR TITLE
Corneliu Rusu (HTP): Updated and added new locales to FOSPN and to Mo…

### PR DESCRIPTION
**Pull request by CorneliuRusu**
…zilla PN

### CODE REVIEW

**Comment by peiying2** [comment]
this is to resolve bugs 1240144 and 1240140

File: firefox_os_privacy_notice/pl.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ System Operacyjny Firefox łączy się z firmą Mozilla oraz innymi podmiotami w
...
-* Pierwsza aktywacja: aby zapewnić dostęp do odpowiednich aktualizacji i aplikacji, System Operacyjny Firefox przy pierwszym włączeniu wysyła jednorazowo do firmy Mozilla pakiet (ping) danych. [Więcej informacji](https://wiki.mozilla.org/FirefoxOS/Metrics)
+* Pierwsza aktywacja: W trakcie wstępnej aktywacji urządzenia z Systemem Operacyjnym Firefox OS wysyła on jednorazowo do firmy Mozilla porcję danych (np. dane związane z urządzeniem, takie jak wersja oprogramowania, identyfikator partnera, kraj i rozmiar ekranu), co pomaga firmie Mozilla określić, czy nasi użytkownicy i partnerzy potrzebują aktualizacji zabezpieczeń i poprawek o znaczeniu krytycznym, , określić wymogi dla programistów aplikacji, by mogli tworzyć lepsze aplikacje dla użytkowników oraz pomóc firmie Mozilla w analizie rynku (np. określeniu liczby aktywowanych urządzeń). [Więcej informacji](https://wiki.mozilla.org/Firefox_OS/Metrics/activationping)
```
Comment (splewako) [inline firefox_os_privacy_notice/pl.md]: What ", , " should mean?

Comment (peiying2) [inline firefox_os_privacy_notice/pl.md]: @splewako, was the above change you made?  Should the extra comma be removed?

**Comment by CorneliuRusu** [comment]
Hi, Peiying,
My colleague, Adrian Ursea, already corrected the issue yesterday. Here is his commit:
https://github.com/mozilla/legal-docs/commit/bdfc4543f22007c19f50605b821b47909cf47a6e

**Comment by splewako** [comment]
Thank you for fixing this.

I just noticed that Mozilla is described as a company "firmie Mozilla" but [not in en-US version](https://github.com/mozilla/legal-docs/blame/master/firefox_os_privacy_notice/en-US.md#L15). I slowly preparing bigger update with fixes for Polish docs, should I also change that as well?

**Pull request merged**